sway/tree/view.c

```diff
@@ -751,6 +751,12 @@ void view_map(struct sway_view *view, struct wlr_surface *wlr_surface,
 		}
 	}
 
+	struct wlr_ext_foreign_toplevel_handle_v1_state foreign_toplevel_state = {0};
+	view->ext_foreign_toplevel =
+		wlr_ext_foreign_toplevel_handle_v1_create(server.foreign_toplevel_list, &foreign_toplevel_state);
```

Comment:
Hm, it's not nice to create the foreign toplevel handle with an empty initial state.

Is it too early to call `view_get_app_id()` here?

Reply:
Testing showed we can create the handle with app_id and title!